cristin-person/src/main/java/no/unit/nva/cristin/person/affiliations/handler/PositionCodesHandler.java

```diff
@@ -35,11 +41,28 @@ public PositionCodesHandler(CristinPositionCodesClient apiClient, Environment en
     protected PositionCodes processInput(Void input, RequestInfo requestInfo, Context context)
         throws ApiGatewayException {
 
-        return apiClient.generateQueryResponse();
+        return apiClient.generateQueryResponse(extractPositionCodeStatusQueryParam(requestInfo));
     }
 
     @Override
     protected Integer getSuccessStatusCode(Void input, PositionCodes output) {
         return HttpURLConnection.HTTP_OK;
     }
+
+    private Boolean extractPositionCodeStatusQueryParam(RequestInfo requestInfo) throws BadRequestException {
+        var activeStatusQueryParam = requestInfo.getQueryParameterOpt(ACTIVE_STATUS_QUERY_PARAM)
+            .map(String::valueOf).orElse(null);
```

Comment:
This is already a String. I think doing `String::valueOf` is unnecessary.

Reply:
This is Optional string so we need to map this to a String value

Reply:
What about:
````
return requestInfo.getQueryParameterOpt(ACTIVE_STATUS_QUERY_PARAM)
    .map(this::validateAndGetPositionCodeStatusBooleanQueryParam).orElse(null);
````

Reply:
Optional gets unwrapped when you do the `orElse`

Reply:
> What about:
> 
> ```
> return requestInfo.getQueryParameterOpt(ACTIVE_STATUS_QUERY_PARAM)
>     .map(this::validateAndGetPositionCodeStatusBooleanQueryParam).orElse(null);
> ```
validateAndGetPositionCodeStatusBooleanQueryParam throws BadRequestException so it's not able to define in method reference inside map

Will Update code as per simply using `orElse `for unwrapping string